candle-nn/src/rnn.rs

```diff
@@ -70,6 +70,12 @@ impl LSTMState {
     }
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum Direction {
+    Forward = 1,
```

Comment:
What is the need for associating integer values to this enum?

Reply:
There actually isn't a need to assign an integer value. I removed it now